### ISSUE TITLE
system stop at startup

### ISSUE THREAD

**Issue by fferraro87** [kuba-moo/mt7630e#13]
sometimes i've this error on startup http://i.imgur.com/BK2GG5K.jpg and i've to shutdown manually my laptop. so what can i do? i've installed your driver with dkms

**Comment by kuba-moo** [comment]
That's weird. Does this issue go away if the driver is removed? Looks like the driver cannot reach the device but USB fails as well for some reason. Can you try booting with `pcie_aspm=off` (it may increase power consumption)?

**Comment by fferraro87** [comment]
yes if i uninstall this driver, the issue go away...it's weird also because it's not every boot but often give this error...now i've boot with this option, if i've this error again, i'll say you. thanks anyway

**Comment by fferraro87** [comment]
nothing...same error also with pcie_aspm=off on boot...
